sermant-integration-tests/dubbo-test/dubbo-2-7-integration-provider/src/main/java/io/sermant/integration/configuration/FlowControlExceptionHandler.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2025-2025 Huawei Technologies Co., Ltd. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.sermant.integration.configuration;
+
+import io.sermant.integration.controller.FlowController;
+
+import org.apache.dubbo.rpc.RpcException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+/**
+ * 流控全局异常处理
+ *
+ * @author chengyouling
```

Comment:
Please use English

Reply:
To test the limiting and circuit breaker capabilities of the server in the dubbo scenario